pyvista/plotting/plotter.py

```diff
@@ -6026,7 +6039,7 @@ def screenshot(
                 # Plotter hasn't been rendered or was improperly closed
                 raise RuntimeError('This plotter is closed and unable to save a screenshot.')
 
-            if self._first_time and not self.off_screen:
+            if self._first_time and (hasattr(self, "off_screen") and not self.off_screen):
```

Comment:
Similar comment

Reply:
I have just added the following comment `# 'off_screen' attribute is specific to Plotter objects.`